h/static/scripts/directive/sidebar-tutorial.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+// @ngInject
+function controller($http, session) {
+  var vm = this;
+
+  vm.showSidebarTutorial = function () {
+    return session.state.show_sidebar_tutorial;
+  };
+
+  vm.dismiss = function () {
+    $http({method: 'POST', url: '/sidebar_tutorial/dismiss'});
+    session.state.show_sidebar_tutorial = false;
```

Comment:
As per Nick's suggestion this has now been changed again, to use an ngResource action